src/app/core/services/auth/auth.service.ts

```diff
@@ -53,7 +53,11 @@ export class AuthService {
       let result = data.result;
       this.localStorage.setLocalData(localKeys.USER_DETAILS, result);
       this.loaderService.stopLoader();
-      this.router.navigate([`/${CommonRoutes.TABS}/${CommonRoutes.HOME}`], { replaceUrl: true });
+      if(result?.user?.hasAcceptedTAndC == true){
+        this.router.navigate([`/${CommonRoutes.TABS}/${CommonRoutes.HOME}`], { replaceUrl: true });
+      } else {
+        this.router.navigate([`/${CommonRoutes.TERMS_AND_CONDITIONS}`]);
```

Comment:
For time being navigate to the home page only here. As backend API is not giving success we will be stuck in this page.

Reply:
ok. but i am redirecting to home page from terms and condition if checkbox is checked.